cmd/livepeer_cli/wizard_bond.go

```diff
@@ -165,8 +165,7 @@ func (w *wizard) bond() {
 
 	amount := big.NewInt(0)
 	for amount.Cmp(big.NewInt(0)) == 0 || balBigInt.Cmp(amount) < 0 {
-		fmt.Printf("Enter bond amount - ")
-		amount = w.readBigInt()
+		amount = w.readBigInt("Enter bond amount - > ")
```

Comment:
I debated this with myself as well. The problem, like you said, is the CLI is not consistent atm. I.e. the bonding amount has `-  > ` as the cursor. But, some other options use only `> `.  E.g. `"Enter new maximum gas price in Wei (enter \"0\" for no maximum gas price) > "`

So, pushing this down into the function would change all of them for this function but then there would be inconsistency between values that use _readBigInt_ and values that use, for instance, _readInt_ or _readString_ and these options are often part of the same sub-menu. I decided to bubble it up so it could be controlled without changing the _readBigInt_ function for this reason (no implicit behavior). The changes preserve existing behavior but if instead, it would be preferable to change the 'standard' prompt to `- >` this will create new inconsistencies. If we want to bubble down the `- > ` I would suggest using only `> `. This would preserve behavior for the current options (they seem to have consistency within a specific sub-menu) at the expense of the global consistency that I think you are alluding to.